csv2bufr/__init__.py

```diff
@@ -19,7 +19,7 @@
 #
 ###############################################################################
 
-__version__ = '0.8.2_dev'
+__version__ = '0.8.2dev'
```

Comment:
Suggest to update versions only as part of a release process.

Reply:
Reverted to 0.8.2_dev